Plugins/Creature/Creature.cpp

```diff
@@ -3393,3 +3394,92 @@ NWNX_EXPORT ArgumentStack BroadcastAttackOfOpportunity(ArgumentStack&& args)
 
     return {};
 }
+
+NWNX_EXPORT ArgumentStack GetMaxSellToStorePriceOverride(ArgumentStack&& args)
+{
+    if (auto *pCreature = Utils::PopCreature(args))
+    {
+        if (auto *pStore = Utils::AsNWSStore(Utils::PopGameObject(args)))
+        {
+            if (auto maxBuyPriceOverride = pCreature->nwnxGet<int32_t>("STORE_MAX_SELL_TO_PRICE_OVERRIDE!" + NWNXLib::Utils::ObjectIDToString(pStore->m_idSelf)))
+            {
+                return maxBuyPriceOverride.value();
+            }
+        }
+    }
+
+    return -2;
+}
+
+NWNX_EXPORT ArgumentStack SetMaxSellToStorePriceOverride(ArgumentStack&& args)
+{
+    static Hooks::Hook pCalculateItemBuyPrice_hook = Hooks::HookFunction(&CNWSStore::CalculateItemBuyPrice,
+    +[](CNWSStore *pThis, CNWSItem *pItem, OBJECT_ID oidSeller) -> int32_t
+    {
+        if (auto *pSeller = Utils::AsNWSCreature(Utils::GetGameObject(oidSeller)))
+        {
+            if (auto maxBuyPriceOverride = pSeller->nwnxGet<int32_t>("STORE_MAX_SELL_TO_PRICE_OVERRIDE!" + NWNXLib::Utils::ObjectIDToString(pThis->m_idSelf)))
+            {
+                if (pItem->m_bPlotObject)
+                    return 0;
+
+                if (auto nCost = pItem->GetCost(true, true, false, false))
```

Comment:
The basegame calls GetCost() using the default args (true, false, false, false), is your second true here intended?

Reply:
Oop, sorry! That'd be a typo on my end.